### DISCUSSION TITLE
Is there a way to ignore local packages in a workspace?

### DISCUSSION THREAD

**Discussion by aladinflux** [pnpm/pnpm#3834] [Q&A]
We have a workspace and we build and publish the modules (all inside `packages` folder) to a private NPM repository. We also have an `app` folder which is part of the workspace.
```yaml
packages:
  - 'packages/*'
  - 'app'
```

When we deploy to production, we'd like the `app` folder to not link packages from the workspace, but download them from `npm`. The issue is, however we run the installer (with filter, with `link-workspace-packages = false`) it still links dependencies from `packages` folder.

How do we tell `pnpm` to download dependencies from remote rather than use local packages?

**Comment by fireairforce** [comment]
Set `link-workspace-packages` as false and in app folder's package.json don't use workspace protocol but the normal version, how about this way?
